MANUAL.txt

```diff
@@ -3377,6 +3377,14 @@ In this example, the document will contain a citation for `item3`
 only, but the bibliography will contain entries for `item1`, `item2`, and
 `item3`.
 
+It is possible to create a bibliography with all the citations, 
+whether or not they appear in the document, by using a wildcard:
+
+    ---
+    nocite: |
```

Comment:
Is this right?  I thought that with the `|` style of YAML quoting, you do not need the single quotes, since the single quotes will be taken as literal.  I think they should be omitted (and of course this should be tested)>